maquetado/scss/_colors.scss

```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
```

Comment:
Remember the structure of the folders that i tell you. `scss/variables/_colors.scss`